conda-recipe/meta.yaml

```diff
@@ -29,20 +29,17 @@ requirements:
     - humanize
     - numpy
     - pytz
+    - boltons
 
 test:
   requires:
     - nslsii_dev_configuration
 
-  # Python imports
   imports:
+    # Import all the packages
     - metadatastore
-    - metadatastore.commands
```

Comment:
The best rationale that I have for dropping these is that importing the **modules** seems like more trouble then its worth.  The main problem that I have had with setup.py is adding a package in the source and not including it in the setup.py so that it is not part of the package.  It is also easy to get the module names in the source out of sync with the module names in this meta.yaml.  The most recent instance of this is that you redid the `Document` object in `doc.py`, not `document.py` but did not update this meta.yaml file to include this update.

Then again, importing every module will **at least** make sure that we don't have import failures from missing deps?  

I could go either way. 
1. Have all modules in the test imports section
2. Have only packages in the test imports section.

What do you want to do?